ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -176,3 +177,45 @@ def test_tensorflow_slogdet(
         fn_tree="linalg.slogdet",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# tensordot
+@given(
+    dtype_x1_x2_axis=_get_dtype_value1_value2_axis_for_tensordot(),
```

Comment:
The `_get_dtype_value1_value2_axis_for_tensordot()` is giving an error as it has a missing required positional argument: `available_dtypes` which causes the tests to fail.